### PR TITLE
Node - Week8 - JWS Basics

### PR DESCRIPTION
**Pull request by AnnaSolovykh**

### CODE REVIEW

File: 05-JWT-Basics/preferred/controllers/main.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const jwt = require('jsonwebtoken');
+
+const logon = async (req, res) => {
+    const { username, password } = req.body;
```
Comment (olearyma) [inline 05-JWT-Basics/preferred/controllers/main.js]: Minor issue. The assignment specified "name" and "password" as the attribute names.
username should be changed to name here.

**Review by olearyma — APPROVED** [review]
Nice work on both assignments.

File: 05-JWT-Basics/preferred/controllers/main.js (hunk 1 above)
Comment (AnnaSolovykh) [reply 05-JWT-Basics/preferred/controllers/main.js]: @olearyma thanks! Made a commit with the required changes.

**Review by olearyma — APPROVED** [review]
Looks good.

**Comment by AnnaSolovykh** [comment]
> Looks good.

Thanks 😊

**Pull request merged**